docs/api/classifier/class_prediction_error.rst

```diff
@@ -7,6 +7,13 @@ The Yellowbrick ``ClassPredictionError`` plot is a twist on other and sometimes
 
 The class prediction error chart provides a way to quickly understand how good your classifier is at predicting the right classes.
 
+=================   ==============================
+Visualizer           :class:`~yellowbrick.classifier.class_prediction_error.ClassPredictionError`
+Quick Method         :func:`~yellowbrick.classifier.class_prediction_error.class_prediction_error`
+Models               Classification
+Workflow             Model evaluation
```

Comment:
Thank you for adding this block to the docs to help orient users to the usage of the ClassPredictionError visualizer; the anchor links look really nice!